response_operations_ui/controllers/edit_contact_details_controller.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from flask import abort
+import requests
+from requests.exceptions import HTTPError
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def edit_contact_details(edit_details_data, respondent_id):
+    logger.debug('Validating contact details')
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/party/update-respondent-details/{respondent_id}'
+
+    response = requests.put(url, json=edit_details_data)
+
+    try:
+        response.raise_for_status()
```

Comment:
Seems a strange way to do this. I think it would be cleaner if it was something along the lines of
```
response = requests.put(url, json=edit_details_data)
if response.status_code == 401:
    abort(401)
```

Reply:
Also needs test coverage